tests/test_mdlp.py

```diff
@@ -32,165 +34,52 @@ def test_find_cut_no_cut():
     k = find_cut(y, 0, len(y))
     assert_equal(-1, k)
 
-def test_mdlp_iris():
-    from sklearn.datasets import load_iris
+def test_e2e():
```

Comment:
Let's rename this to `test_fit_transform`

Reply:
👍